packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Skeleton, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {useIntl} from 'react-intl'
+import {useCurrency} from '@salesforce/retail-react-app/app/hooks'
+
+const DisplayPrice = ({
+    basePrice,
+    discountPrice,
+    isProductASet = false,
+    currency,
+    discountPriceProps,
+    basePriceProps,
+    skeletonProps
+}) => {
+    const intl = useIntl()
+    const {currency: activeCurrency} = useCurrency()
+    return (
+        <Skeleton isLoaded={basePrice} display={'flex'} {...skeletonProps}>
+            <Text fontWeight="bold" fontSize="md" aria-label="price" mr={1}>
```

Comment:
```suggestion
            <Text fontWeight="bold" fontSize="md" mr={1}>
```
A text field doesn't need an aria-label.